docs/setup.md

```diff
@@ -2,11 +2,15 @@
 Clone the kiva-pricing-tool repository onto your computer.
 
 ## Frontend
+If you don't node-gyp installed globally, do so now
```

Comment:
Maybe you'd like to change the line to  "if you don't have"? @amychan331

Reply:
Oops... missing word now fixed!